src/vfd/sriov.c

```diff
@@ -272,6 +307,28 @@ tx_vlan_insert_set_on_vf(portid_t port_id, uint16_t vf_id, int vlan_id)
 	}
 }
 
+void
+tx_cvlan_insert_set_on_vf(portid_t port_id, uint16_t vf_id, int vlan_id)
+{
+	int diag = 0;
+			
+	uint dev_type = get_nic_type(port_id);
+	switch (dev_type) {
+		case VFD_MLX5:
+			diag = vfd_mlx5_set_vf_cvlan_insert( port_id, vf_id, vlan_id );
+			break;
+			
+		default:
+			bleat_printf( 0, "tx_vlan_insert_set_on_vf: unknown device type: %u, port: %u", port_id, dev_type);
```

Comment:
This message will be confusing especially since it will be followed by a successful message generated below.  Maybe the message "device does not support cvlan insert; no action taken" and then returning rather than falling out to issue the successful message.

Reply:
This is actually true for all other actions, except they have a "case" for every possible device u work with.
How about maintaining the same flow as in other operations?
I will add a case for the other 3 devices with no action taken. This way the "unknown device" won't be printed and only the successful message will appear.

Reply:
That is just fine, and makes perfect sense.   Thank you.